src/components/Card/CardHeader/index.test.js

```diff
@@ -0,0 +1,46 @@
+import CardHeader from '.';
+import { screen, fireEvent, within } from '@testing-library/react';
+import { renderWithRouter } from '../../../tests/Helpers/renderWithRouter';
+
+const mockSetHeader = jest.fn();
+const mockSetEditApproved = jest.fn();
+const mockReduxDispatch = jest.fn();
+
+jest.mock('react-redux', () => ({
+  ...jest.requireActual('react-redux'),
+  useDispatch: () => mockReduxDispatch,
+}));
+
+test('card header should be rendered', () => {
+  renderWithRouter(
+    <CardHeader
+      editApproved={true}
+      setHeader={mockSetHeader}
+      setEditApproved={mockSetEditApproved}
+      header="header"
+    />
+  );
+  expect(screen.getByTestId('cardHeader')).toBeInTheDocument();
```

Comment:
Check not only that the component is on the page, but also that it contains the right value